src/gui/folder.cpp

```diff
@@ -64,6 +64,18 @@ auto versionC()
 
 constexpr int WinVfsSettingsVersion = 4;
 constexpr int SettingsVersion = 2;
```

Comment:
Thats the version written to the settings so you need to bump both to 5 I assume or better unify them.

Reply:
Where is the other one?

Reply:
done

Reply:
the two lines above.

Reply:
I don't think it's needed, see the commit message.

Reply:
I think the uiid was only used when with Windows vfs disabled, with vfs we use a different mechanism.
So you need to bump the non vfs version too.
Thats why I suggested to merge both and only use one version independent of vfs.

Reply:
I merged them. Please check if I did it correctly.